.github/workflows/ci-pipline.yml

```diff
@@ -11,6 +11,17 @@ on:
       - 'documentation/**'
       - 'README.md'
 
+env:
+  TF_VAR_prefix: 'aliwen-gh'
```

Comment:
Bonus point - you've run this with a couple of different prefixes I think (e.g. `aliwen-2` at some point) which has generated a couple of bonus web apps, but I suspect you're using the same GitHub client id/client secret for all of them so they're all redirecting to the same one. It would be good to ensure you have different credentials for each so that they're truly stand-alone!